core/core.go

```diff
@@ -41,7 +41,7 @@ type ServiceInfo struct {
 // comunicate.
 type Network interface {
 	CreateNetwork(id string) error
-	DeleteNetwork(id string) error
+	DeleteNetwork(id, encap string, pktTag, extPktTag int) error
```

Comment:
We need encap/pktTag to cleanup OVS state. problem here was, when etcd sends delete notification, object is gone. So, only way to get the state back is using an oper state. I want to avoid keeping duplicate info in opera state. So, only way was to pass it down the API. But, thinking about this. Now that I've moved network/endpoint state into mastercfg, better way might be to pass the state itself plugin API calls. I'll file an issue to pick this up in next round of cleanup.